test/input_files/sa_2d_turbulent_channel_regression.in

```diff
@@ -0,0 +1,107 @@
+# Mesh related options
+[mesh-options]
+mesh_option = create_2D_mesh
+element_type = QUAD4
+mesh_nx1 = 20
+mesh_nx2 = 150
+redistribute = '{x}{0.5*(1 + (tanh((y -0.5)*3.0)/tanh(1.5)))}{z}'
+domain_x1_max = 5.0
+domain_x2_max = 1.0
+
+# Options for time solvers
+[unsteady-solver]
+transient = false 
+
+#Linear and nonlinear solver options
+[linear-nonlinear-solver]
+max_nonlinear_iterations = 25
+max_linear_iterations = 50000
+
+initial_linear_tolerance = 1.0e-12
```

Comment:
This line was nonsense when it was written (the initial linear tolerance can't go below the minimum linear tolerance, which defaults to like 1e-3) and I find it tragically amusing how many subsequent input files it's been copied into.